reframe/core/schedulers/slurm.py

```diff
@@ -28,11 +27,15 @@ def slurm_state_completed(state):
         'PREEMPTED',
         'TIMEOUT',
     }
-    return state in completion_states
+    if state:
+        return all(s in completion_states for s in state.split(','))
```

Comment:
Why not use directly the `util.allx()`?

Reply:
Because I cannot use `split` if `state` is `None`